epregressions/tests/resources/dummy.energyplus.py

```diff
@@ -22,7 +23,10 @@
 import json
 import sys
 
-with open('in.idf') as f_idf:
+file_name = 'in.idf'
+if os.path.exists('in.epJSON'):
+    file_name = 'in.epJSON'
+with open(file_name) as f_idf:
```

Comment:
Had to modify the dummy EnergyPlus script to allow it to recognize a possible `in.epJSON` file.